src/main/java/algorithms/curated170/medium/FindPermutation1.java

```diff
@@ -0,0 +1,45 @@
+package algorithms.curated170.medium;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class FindPermutation1 {
+    public static int[] findPermutation(String s) {
+        s = s + ".";
```

Comment:
Aslında burada s Stringine fazladan bir karakter eklemek hoşuma girmiyor ama, while döngüsüne en azından 1 kere girebilmesi ve "I" gibi tek bir karakterden oluşan "s"'lerde hatâ vermemesi için bu şart görünüyor.